lib/matplotlib/axes/_axes.py

```diff
@@ -4533,7 +4533,7 @@ def barbs(self, *args, **kw):
 
     # Uses a custom implementation of data-kwarg handling in
     # _process_plot_var_args.
-    def fill(self, *args, **kwargs):
+    def fill(self, *args, data=None, **kwargs):
```

Comment:
`data` should be added to the Parameters section in the docstring

Reply:
done